cloud/pkg/cloudstream/containermetrics_connection.go

```diff
@@ -74,8 +74,9 @@ func (ms *ContainerMetricsConnection) SendConnection() (stream.EdgedConnection,
 		URL:    *ms.r.Request.URL,
 		Header: ms.r.Request.Header,
 	}
+	targetPort := strings.Split(ms.r.Request.Host, ":")[1]
 	connector.URL.Scheme = "http"
-	connector.URL.Host = net.JoinHostPort("127.0.0.1", fmt.Sprintf("%v", constants.ServerPort))
+	connector.URL.Host = net.JoinHostPort("127.0.0.1", fmt.Sprintf("%v", targetPort))
```

Comment:
Still need to change edge streamserver config.

Reply:
no need change  edge streamserver config.
For this implementation, we can support multiple ports.

Fox exmaple, if add two iptables rule like
```
DNAT       tcp  --  0.0.0.0/0            0.0.0.0/0            tcp dpt:10350 to:192.168.96.94:10003
DNAT       tcp  --  0.0.0.0/0            0.0.0.0/0            tcp dpt:9100 to:192.168.96.94:10003
```
then,
```
curl  -k  https://edgenode1:10350/metrics // will forward to  edgenode:10350
curl  -k  https://edgenode1:9100/metrics // will forward to  edgenode:9100 (that is node_exporter)
```

Reply:
The common process is that prometheus use SVC to connect node_exporter and scrape.

After we support SVC between edge and cloud, it still works.

So is it necessary?

/cc @fisherxu @kevin-wangzefeng

Reply:
"After we support SVC between edge and cloud, it still works." 
When can we support SVC between edge and cloud? Seems we still more than half a year :(